### PR TITLE
network logs support key in lt config

### PR DESCRIPTION
**Pull request by asad9711**
https://lambdatest.atlassian.net/browse/MLE-2617

### CODE REVIEW

File: commands/utils/default_config.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ module.exports={
         "specs":"./*.spec.js",
         "ignore_files":"",
         "feature_file_suppport": false,
+        "network": false,
```
Comment (japneetlambdatest) [inline commands/utils/default_config.js:29]: We will have to handle this in the set args file to handle the defaults

Comment (japneetlambdatest) [reply commands/utils/default_config.js:29]: Also add it in index file to handle the params from Command line

Comment (asad9711) [reply commands/utils/default_config.js:29]: resolved

File: commands/utils/set_args.js [hunk 2]
```diff
@@ -227,6 +227,14 @@ function sync_args_from_cmd(args) {
     } else {
       lt_config["tunnel_settings"]["autostart"] = true;
     }
+   
+    if ("network" in args) {
+      lt_config["run_settings"]["network"] = true
+        ? args["network"] == "true"
+        : false;
+    } else if (!lt_config["run_settings"]["network"]) {
+      lt_config["run_settings"]["network"] = false;
```
Comment (japneetlambdatest) [inline commands/utils/set_args.js:236]: Please add validation to validate this in validate.js
that the value is either true or false

Comment (asad9711) [reply commands/utils/set_args.js:236]: resolved

**Pull request merged**